src/app/api/statistics/route.js

```diff
@@ -5,59 +5,64 @@ import { authenticate } from "@/utils/auth";
 
 export const GET = async () => {
   const res = NextResponse;
-  const { auth } = await authenticate({
-    admins: [1],
-  });
 
-  if (auth !== 200) {
-    return res.json(
-      { message: `Authentication Error: ${"MESSAGE VARIABLE SHOULD BE HERE"}` },
-      { status: auth }
-    );
-  }
+  return authenticate({ admins: [1] })
+    .then(({ auth }) => {
+      if (auth !== 200) {
+        return res.json(
+          {
+            message: `Authentication Error: ${"MESSAGE VARIABLE SHOULD BE HERE"}`,
+          },
+          { status: auth }
+        );
+      }
 
-  try {
-    const {
-      teams: { 1: teams },
-      participants: { 1: participants },
-      volunteers: { 1: volunteers },
-      judges: { 1: judges },
-      mentors: { 1: mentors },
-      committees: { 1: committees },
-      sponsors: { 1: sponsors },
-      panels: { 1: panels },
-      admins: { 1: admins },
-    } = (await getDoc(doc(db, "statistics", "statistics"))).data();
+      const statisticsPromise = getDoc(doc(db, "statistics", "statistics"));
+      const eventsPromise = getDocs(collection(db, "events"));
 
-    const events = await getDocs(collection(db, "events"));
+      return Promise.all([statisticsPromise, eventsPromise]);
+    })
+    .then(([statisticsSnapshot, eventsSnapshot]) => {
```

Comment:
keep it await/async instead of then catch, then catch will primarily be used for frontend and backend will be async await